content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -719,6 +719,14 @@ Used on: Namespace
 
 This annotation requires the [PodTolerationRestriction](/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction) admission controller to be enabled. This annotation key allows assigning tolerations to a namespace and any new pods created in this namespace would get these tolerations added.
 
+### scheduler.alpha.kubernetes.io/tolerationsWhitelist {#schedulerkubernetestolerations-whitelist}
+
+Example: `scheduler.alpha.kubernetes.io/tolerationsWhitelist: '[{"operator": "Exists", "effect": "NoSchedule", "key": "dedicated-node"}]'`
+
+Used on: Namespace
+
+This annotation also requires the [PodTolerationRestriction](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction) admission controller to be enabled. This annotation key adds lists of allowed tolerations to the namespace. In case of any conflict between tolerations of a pod and the tolerations of its namespace, the tolerations of the pod is checked against this list. The pod request is admitted only if the check succeeds.
```

Comment:
nit from docs.

```suggestion
This annotation requires the [PodTolerationRestriction](/docs/reference/access-authn-authz/admission-controllers/#podtolerationrestriction) admission controller to be enabled. This annotation key adds lists of allowed tolerations to the namespace. In case of any conflict between tolerations of a pod and the tolerations of its namespace, the tolerations of the pod is checked against this list. The pod request is admitted only if the check succeeds.
```

Reply:
Hm. I missed the link , thanks for the review @divya-mohan0209